enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1220 @@
+> timezones in this delay as they are not known and irrelevant to my immediate concern."
+
+> "As a cluster lifecycle administrator, I want to ensure any material changes to my 
+> control-plane are only initiated during well known windows of low service utilization to 
+> reduce the impact of any service disruption. Furthermore, I want to ensure that material
+> changes to my worker-nodes occur on a less frequent cadence because I know my workloads
+> are not HA."
+
+> "As an SRE, tasked with performing non-emergency corrective action, I want 
+> to be able to apply a desired configuration (e.g. PID limit change) and have that change roll out 
+> in a minimally disruptive way subject to the customer's configured maintenance schedule."
+
+> "As an SRE, tasked with performing emergency corrective action, I want to be able to 
+> quickly disable a configured maintenance schedule, apply necessary changes, have them roll out immediately, 
+> and restore the maintenance schedule to its previous configuration."
+
+> "As a leader within the Service Delivery organization, tasked with performing emergency corrective action
+> across our fleet, I want to be able to bypass and then restore customer maintenance schedules
+> with minimal technical overhead."
+
+> "As a cluster lifecycle administrator who is well served by a fully managed update without change management, 
+> I want to be minimally inconvenienced by the introduction of change management / maintenance schedules."
+
+> "As a cluster lifecycle administrator who is not well served by a fully managed update and needs exacting 
+> control over when material changes occur on my cluster where opportunities do NOT arise at reoccurring intervals,
+> I want to employ a change management strategy that defers material changes until I perform a manual action."
+
+> "As a cluster lifecycle administrator, I want to easily determine the next time at which maintenance operations
+> will be permitted to be initiated, based on the configured maintenance schedule, by looking at the 
+> status of relevant API resources or metrics."
+
+> "As a cluster lifecycle administrator, I want to easily determine whether there are material changes pending for
+> my cluster, awaiting a permitted window based on the configured maintenance schedule, by looking at the 
+> status of relevant API resources or metrics."
+
+> "As a cluster lifecycle administrator, I want to easily determine whether a maintenance schedule is currently being
+> enforced on my cluster by looking at the status of relevant API resources or metrics."
+
+> "As a cluster lifecycle administrator, I want to be able to alert my operations team when changes are pending,
+> when and the number of seconds to the next permitted window approaches, or when a maintenance schedule is not being
+> enforced on my cluster."
+
+> "As a cluster lifecycle administrator, I want to be able to diagnose why pending changes have not been applied
+> if I expected them to be."

```

Comment:
I will have to read the node reboot policy EP, but certainly MCO pausing is within the scope of what the operator could interpret and report up through status for maintenance schedule.

Reply:
Resolving this thread since the node disruption policy EP seems complementary with what is proposed with maintenance schedules. Details added in the `Node Disruption Policy` section.